components/TopNav.vue

```diff
@@ -50,12 +49,20 @@
             </nav>
             <LocaleSelector />
         </div>
+        <HamburgerComponent v-else />
     </div>
 </template>
 
 <script lang="ts" setup>
 import { useSubmissionStore } from "~/stores/submissionStore";
 import { useMenuStore } from "~/stores/menuStore";
+import { useNuxtApp } from "#app";
```

Comment:
I don't think we need this import here

Reply:
@ermish It's needed for `$viewport`:

`const { $viewport } = useNuxtApp();`